ebuku.el

```diff
@@ -853,6 +853,49 @@ If an argument is excluded, get it from `ebuku-cache-default-args'."
       (setq tags (nconc (map-elt bookmark 'tags) tags)))
     (setq ebuku-tags (sort (seq-uniq tags) 'string-collate-lessp))))
 
+(defun ebuku-append-tag-to-bookmark ()
+  "Interactive function to append tag to bookmark."
+  ;; TODO fix an issue where it moves the cursor to the top after adding
```

Comment:
i think i might need to put a `save-excursion` inside `ebuku-refresh`. Could you please remove this line, and i'll add it to my TODO list.

Reply:
Removed.